src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -0,0 +1,477 @@
+                   i18n>
+              Crush ruleset
+            </label>
+            <div class="col-sm-9"
+                 [popover]="crushSteps"
+                 popoverTitle="Steps"
+                 triggers="mouseenter:mouseleave">
+              <select class="form-control"
+                      id="crushSet"
+                      formControlName="crushRule"
+                      name="crushSet">
+                <option i18n
+                        [ngValue]="null">
+                  -- Select a crush rule --
+                </option>
+                <option *ngFor="let rule of current.rules"
+                        [ngValue]="rule">
+                  {{ rule.rule_name }}
+                </option>
+              </select>
+              <span class="help-block"
+                    i18n
+                    *ngIf="form.showError('crushRule', formDir, 'tooFewOsds')">
+                The rule can't be used in the current cluster as it has to few OSDs to meet the
+                minimum required OSD by this rule.
+              </span>
+            </div>
+          </div>
+
+          <!-- Replica Size -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('size', formDir)}"
+               *ngIf="form.getValue('poolType') === 'replicated'">
+            <label i18n
+                   class="control-label col-sm-3"
+                   for="size">
+              Replicated size
+              <span class="required"></span>
+            </label>
+            <div class="col-sm-9">
+              <input class="form-control"
+                     id="size"
+                     [max]="getMaxSize()"
+                     [min]="getMinSize()"
+                     name="size"
+                     type="number"
+                     formControlName="size">
+              <span class="help-block"
+                    *ngIf="form.showError('size', formDir)">
+                <ul class="list-inline">
+                  <li i18n>
+                    Minimum: {{ getMinSize() }}
+                  </li>
+                  <li i18n>
+                    Maximum: {{ getMaxSize() }}
+                  </li>
+                </ul>
+              </span>
+              <span class="help-block"
+                    i18n
+                    *ngIf="form.showError('size', formDir)">
+                The size specified is out of range.
+                A value from {{ getMinSize() }} to {{ getMaxSize() }} is valid.
+              </span>
+            </div>
+          </div>
+
+          <!-- Erasure Profile select -->
+          <div class="form-group"
+               *ngIf="form.getValue('poolType') === 'erasure'">
+            <label i18n
+                   class="control-label col-sm-3"
+                   for="erasureProfile">
+              Erasure code profile
+            </label>
+            <div class="col-sm-9">
+              <!-- <div class="input-group"> -->
+              <div>
```

Comment:
Comment (L233) and container without attributes (L234) may not be required here.